### ISSUE TITLE
Can a single pin be used for trasmitting/receiving? Works at 1mz?

### ISSUE THREAD

**Issue by Rimbaldo** [mihaigalos/Drivers#5]
Can it be set to tx/rx with only one pin on each attiny (besides gnd)?

To reduce power draw on the attiny85, at which speeds will it get to send a single character at 1mhz?

**Comment by mihaigalos** [comment]
Hi @Rimbaldo,
Yes, a single pin can be used for both Tx and Rx, even at 1Mhz. You will however need a software UART (bit-bang). See [this](https://github.com/mihaigalos/Drivers/tree/master/AVR/SoftwareUart).
I believe you can go to 9600 bps easily, surely also 38400 bps.

**Comment by Rimbaldo** [comment]
Thanks... do you have an  arduino compatible library of yours?

**Comment by mihaigalos** [comment]
Yes. It is even referenced in my previous message. Click on the blue _this_ there.

**Comment by Rimbaldo** [comment]
I clicked! But I couldn't find in the examples any .INO files. I thought it was pure C++, not meant  to be used with the arduino IDE. BUt if it is, I can give it a try!

**Comment by Rimbaldo** [comment]
Hi! I’ve looked into your lib, but I have some doubts. Can I use a single pin from each MCU wired directly between each other,  or do I need two pins in each MCU and that transistor/resistor diagram to get only one line to “transceive” between two Attinys?

**Comment by mihaigalos** [comment]
One single pin can be used on each side. You need to adapt the code for toggling between Rx and Tx.
